ros2launch/ros2launch/command/launch.py

```diff
@@ -73,8 +73,12 @@ class LaunchCommand(CommandExtension):
     def add_arguments(self, parser, cli_name):
         """Add arguments to argparse."""
         parser.add_argument(
-            '-d', '--debug', default=False, action='store_true',
-            help='Put the launch system in debug mode, provides more verbose output.')
+            '-d',
+            '--debug',
+            default=False,
+            action='store_true',
+            help='Put the launch system in debug mode, provides more verbose output.',
+        )
```

Comment:
I don't see any real modifications here, it probably shouldn't be in the diff.

Reply:
Accidentally left this when trying to autoformat my code with black, will remove